micro-ui/web/micro-ui-internals/packages/modules/sandbox/src/pages/employee/ProductPage/index.js

```
@@ -21,12 +18,32 @@ const ProductPage = () => {
+    "BND",
+    "FSM",
+    "Firenoc",
+    "SURVEY",
+  ];
 
+  const sortedDetails = React.useMemo(() => {
+    if (!detailsConfig) return [];
+    return [...detailsConfig].sort((a, b) => {
+      const indexA = preferredOrder.indexOf(a.module);
+      const indexB = preferredOrder.indexOf(b.module);
+      return (indexA === -1 ? Infinity : indexA) - (indexB === -1 ? Infinity : indexB);
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)